autofix-go/replace-to-replaceAll.batch.yml

```
@@ -0,0 +1,18 @@
+name: replace-to-replaceAll
+description: |
+  This batch change uses [Comby](https://comby.dev) to replace the deprecated String.replaceAll(a,b,c,-1) by String.Replace(a,b,c)
+
+on:
+  - repositoriesMatchingQuery: context:@malo/sourcegraph .Replace(..., ..., -1) lang:go patternType:structural
+
+steps:
+  - run: comby -in-place '.Replace(:[a],:[b],:[c],...)' '.ReplaceAll(:[a],:[b],:[c])' .go -matcher .go -exclude-dir .,vendor
```

```suggestion
  - run: comby -in-place '.Replace(:[a],:[b],:[c], -1)' '.ReplaceAll(:[a],:[b],:[c])' .go -matcher .go -exclude-dir .,vendor
```

I think using `...` would make false-positives like `strings.Replace(x, y, z, 1)`.